Neos.Neos/Classes/Service/EditorContentStreamZookeeper.php

```diff
@@ -102,48 +101,39 @@ public function relayEditorAuthentication(Authentication\TokenInterface $token):
                 break;
             }
         }
-
-        if ($isEditor) {
-            $user = $this->partyService->getAssignedPartyOfAccount($token->getAccount());
-            if ($user instanceof User) {
-                $workspaceName = AdjustmentsWorkspaceName::fromAccountIdentifier(
-                    $token->getAccount()->getAccountIdentifier()
-                );
-                $workspace = $contentRepository->getWorkspaceFinder()->findOneByName(
-                    $workspaceName->toContentRepositoryWorkspaceName()
-                );
-
-                if (!$workspace) {
-                    // @todo: find base workspace for user
-                    /** @var Workspace $baseWorkspace */
-                    $baseWorkspace = $contentRepository->getWorkspaceFinder()->findOneByName(WorkspaceName::forLive());
-                    $editorsNewContentStreamId = ContentStreamId::create();
-                    $similarlyNamedWorkspaces = $contentRepository->getWorkspaceFinder()->findByPrefix(
-                        $workspaceName->toContentRepositoryWorkspaceName()
-                    );
-                    if (!empty($similarlyNamedWorkspaces)) {
-                        $workspaceName = $workspaceName->increment($similarlyNamedWorkspaces);
-                    }
```

Comment:
see slack discussion about this odd logic https://neos-project.slack.com/archives/C3MCBK6S2/p1695120487251679

Reply:
Do we assume now that we can indeed delete this part?

@nezaniel I'm tagging you here again as I think the slack conversation was missed 😉

Reply:
Yes after thinking it through together with basti we are positive that this case should never occur.

But to get full reassurance we could place a logger/debug statement here and observe this behaviour in a real world project - maybe you could help out with this? ^^

Reply:
> Yes after thinking it through together with basti we are positive that this case should never occur.

I'm with you on this. It's just that going through this MR and trying to figure out if we are on the same page on this is difficult when we have this unresolved comment and an unanswered question in Slack 😉.

Eventually leading to merge request settling dust while they would otherwise (maybe) have been merged.

Reply:
Sorry for "unresolving" this. I totally agree, we should try to get this one in asap as we can always (re)add missing features.
But I would like to have a quick 👍🏻 or 👎🏻  from @nezaniel and/or @skurfuerst on this one, so I added it to our ESCR weekly agenda – won't take long

Reply:
I tested a lot of things but couldnt find a case where this logic will be used. Instead if found other odd things: https://github.com/neos/neos-development-collection/issues/4566

On thing we could do: 

check in our client projects if `flow workspace:list | grep user-` has an incremented number.

Reply:
In neos 8.x there was actually never this incrementing logic thats why i couldnt find an "incremented" workspace in a bigger client project.

the workspace was created when the user was added:

https://github.com/neos/neos-development-collection/blob/072186e1601d82a8565942a05c9fe7253cf7f85d/Neos.Neos/Classes/Domain/Service/UserService.php#L876

or dynamically in `context->getWorkspace`
https://github.com/neos/neos-development-collection/blob/83ef0da8cc960503f85201590d9728dadc6b01f4/Neos.ContentRepository/Classes/Domain/Service/Context.php#L157-L162

Reply:
we discussed this with Bernhard and he also saw no reason for this logic.